src/views/Editor.vue

```diff
@@ -542,10 +555,19 @@ export default {
 				await this.openRecord(this.id)
 			}
 		},
+		'qvainData.next': {
+			handler(newNext) {
+				const newBool = !!newNext
```

Comment:
Isn't 2 time negate is the same value again?

Reply:
Yes, if the original value was a boolean, which is not the case here. Double negation is used in Javascript to convert things to booleans. `!newNext` converts the value to a boolean and negates it, and the second `!` negates the negation so you get the original value converted into a boolean. You could also use `Boolean(newNext)` to get the same result, but `!!` is pretty common in js.